app/services/websites/delete.rb

```diff
@@ -0,0 +1,42 @@
+module Websites
+  class Delete
+    def self.call(params, user)
+      new(params, user).call
+    end
+
+    def initialize(params, user)
+      @params = params
+      @user = user
+    end
+
+    def call
+      delete_website
+      update_statistics
+      find_or_create_user
+    end
+
+    private
+
+    attr_reader :params, :user, :website
+
+    def delete_website
+      CollectionWebsite.where(collection_id: collection_id).where(website_id: params[:id]).first.destroy
+    end
```

Comment:
@redtachyons I think destroy_all should be fine here, since there's going to be only one entry anyway. I dislike using `find` or `first` here, because they execute a query, load the object into memory and then run the delete query.